modules/ROOT/pages/_partials/mmp-concept.adoc

```diff
@@ -108,16 +108,18 @@ For example:
 
 // tag::businessGroupParameterDescription[]
 | `businessGroup` | The Business group path of the deployment +
-For example:
+Specify the full hierarchical path from the parent organization to the target Business group, for example:
 [source,xml,linenums]
 ----
-<businessGroup>MasterOrg\SubOrg</businessGroup>
+<businessGroup>ParentOrg\SubOrg1\myBusinessGroup</businessGroup>
 ----
 | No
 // end::businessGroupParameterDescription[]
 
 // tag::businessGroupIdParameterDescription[]
 | `businessGroupId` | The Business group id of the deployment +
+Instead of specifying the Business group path, you can specify the Business group id to deploy your application. +
```

Comment:
"id" -> "ID" 
Prob should be in our term reference. See https://www.merriam-webster.com/dictionary/id